tmt/utils/__init__.py

```diff
@@ -230,13 +230,16 @@ def format(cls, exit_code: int) -> Optional[str]:
 T = TypeVar('T')
 
 
-def effective_workdir_root() -> Path:
+def effective_workdir_root(workdir_root_option: Optional[str] = None) -> Path:
     """
     Find out what the actual workdir root is.
 
-    If ``TMT_WORKDIR_ROOT`` variable is set, it is used as the workdir root.
-    Otherwise, the default of :py:data:`WORKDIR_ROOT` is used.
+    If ``workdir-root`` cli option is set, it is used as the workdir root.
+    Otherwise, ``TMT_WORKDIR_ROOT`` variable is used, if it is set. Or, the default
+    of :py:data:`WORKDIR_ROOT` is used.
```

Comment:
```suggestion
    If the `workdir-root` cli option is set, it is used as the workdir root.
    Otherwise, the ``TMT_WORKDIR_ROOT`` environment variable is used if set.
    If neither is specified, the default value of :py:data:`WORKDIR_ROOT` is used.
```

Reply:
@martinhoyer why a single backtick btw?

Reply:
Sorry I missed this comment. The double to single quotes were not intentional change. I'm just suggesting the wording changes.